jac-uk/admin
# Fix qualification update

## What's included?
Dodgy logic was doing some odd merging of data when updating an candidates qualification section of their application, adding it to the root level as well as updating the qualifications. 
This should resolve that. 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- make an update to an applications qualification section.  ( The bug was only present when updating - not creating a new item in the list or removing!)
- Check firebase to ensure the data model is correct (ie. not being added to the root level of the application document)

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

> example of bad data generated prior to fix
![image](https://user-images.githubusercontent.com/44227249/154734488-9c1b884f-e53b-48f9-ba0b-89dd0ad976ef.png)

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
